README.md

```diff
@@ -123,6 +123,12 @@ To manually run RCCL with NPKit enabled, environment variable `NPKIT_DUMP_DIR` n
 
 To manually analyze NPKit dump results, please leverage [npkit_trace_generator.py](https://github.com/microsoft/NPKit/blob/main/rccl_samples/npkit_trace_generator.py).
 
+## MSCCL++
+RCCL integrates MSCCL++ (https://github.com/microsoft/mscclpp) to leverage the highly efficient GPU-GPU communication primitives for collective operations. Thanks to Microsoft Corporation for collaborating with us in this project.
+
+RCCL can be built with MSCCL++ support by providing the `-ENABLE_MSCCLPP` flag during compilation. RCCL allreduce and allgather collectives can leverage the efficient MSCCL++ communication kernels for certain message sizes. Users need to set the RCCL environment variable `RCCL_ENABLE_MSCCLPP=1` to run RCCL workload with MSCCL++ support.
```

Comment:
The flag should be `-DENABLE_MSCCLPP=ON`. Also, should we document `RCCL_MSCCLPP_THRESHOLD`?

Reply:
maybe we should preface by saying use `--enable-mscclpp` when using the `./install.sh` script\
and, use `-DENABLE_MSCCLPP=ON` if using CMake directly

Reply:
I don't think we have similarly complete docs around the other options available in `./install.sh`, `cmake`, or `make`. Perhaps we should, but for this, maybe we only just need to be consistent?

Reply:
You're right, we should add it above.

Reply:
yeah, currently, only the `./install.sh --help` output is listed in README.md to show available options...\
we could extend this with more info in the README
P.S: no `make` for RCCL